ravenframework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -169,192 +198,266 @@ def _evaluate(self, datasets, **kwargs):
     """
     realizations = []
     realizationArray = []
-    for feat, targ, scaleRatioBeta, scaleRatioOmega in zip(self.features, self.targets, self.scaleRatioBeta, self.scaleRatioOmega):
-      nameFeat = feat.split("|")
-      nameTarg = targ.split("|")
-      names = [nameFeat[0],nameTarg[0]]
-      featData = self._getDataFromDatasets(datasets, feat, names)[0]
-      targData = self._getDataFromDatasets(datasets, targ, names)[0]
-      if (isinstance(scaleRatioBeta,int) or isinstance(scaleRatioBeta,float)) and (isinstance(scaleRatioOmega,int) or isinstance(scaleRatioOmega,float)) is True:
-        if self.scaleType == 'DataSynthesis':
-          timeScalingRatio = 1
-          if abs(1-scaleRatioBeta) > 10**(-4) or abs(1-scaleRatioOmega) > 10**(-4):
-            self.raiseAnError(IOError, "Either beta or omega scaling ratio are not 1. Both must be 1")
-        elif self.scaleType == '2_2_affine':
-          timeScalingRatio = scaleRatioBeta/scaleRatioOmega
-        elif self.scaleType == 'dilation':
-          timeScalingRatio = 1
-          if abs(1-scaleRatioBeta/scaleRatioOmega) > 10**(-4):
-            self.raiseAnError(IOError, "Beta scaling ratio:",scaleRatioBeta,"and Omega scaling ratio:",scaleRatioOmega,"are not nearly equivalent")
-        elif self.scaleType == 'beta_strain':
-          timeScalingRatio = scaleRatioBeta
-          if abs(1-scaleRatioOmega) > 10**(-4):
-            self.raiseAnError(IOError, "Omega scaling ratio:",scaleRatioOmega,"must be 1")
-        elif self.scaleType == 'omega_strain':
-          timeScalingRatio = 1/scaleRatioOmega
-          if abs(1-scaleRatioBeta) > 10**(-4):
-            self.raiseAnError(IOError, "Beta scaling ratio:",scaleRatioBeta,"must be 1")
-        elif self.scaleType == 'identity':
-          timeScalingRatio = 1
-          if abs(1-scaleRatioBeta) > 10**(-4) or abs(1-scaleRatioOmega) > 10**(-4):
-            self.raiseAnError(IOError, "Either beta or omega scaling ratio are not 1. Both must be 1")
-        else:
-          self.raiseAnError(IOError, "Scaling Type",self.scaleType, "is not provided")
+    if len(self.features) > 1 or len(self.targets) > 1:
+      self.raiseAnError(IOError, "The number of inputs for features or targets is greater than 1. Please restrict to one set per step.")
```

Comment:
In the manual, you mentioned the extra features/targets will be ignore, here an error will be raised. Please update the manual to reflect this error.

Reply:
Manual now mentions error output.